lib/request/TingClientSearchRequest.php

```diff
@@ -213,6 +213,12 @@ public function processResponse(stdClass $response) {
     $searchResult->numTotalCollections = self::getValue($searchResponse->result->collectionCount);
     $searchResult->more = (strcasecmp('true', self::getValue($searchResponse->result->more)) == 0);
 
+    $error = isset($searchResponse->error) ? self::getValue($searchResponse->error) : NULL;
+    if(isset($error)){
+      $searchResult->error = $error;
+    }
```

Comment:
This change makes little sense to me. Right above this we check for the same `error` property and throw an exception if it is present. Thus we should never be able to reach this part of the code.